Makefile

```diff
@@ -240,6 +246,10 @@ check_fmt_js: check_nvm_installed
 	nvm use $(NODE_VERSION) && \
 	$(MAKE) -C tfhe/web_wasm_parallel_tests check_fmt
 
+.PHONY: check_typos # Check for typos in codebase
+check_typos: install_typos_checker
+	@typos && echo "No typos found"
```

Comment:
if typos are detected exit code is non zero ?

Reply:
indeed